### PR TITLE
Added home page

### PR DESCRIPTION
**Pull request by vrinda-mahajan**
In this PR I have added the home page with category filter functionality.

Files to be checked:
-src/components/card/video-card/video-card.jsx
-src/components/card/video-card/video-card.css

-src/components/chip-container/chip-container.jsx
-src/components/chip-container/chip-container.css

-src/components/navbar/navbar.jsx
-src/components/navbar/navbar.css

-src/components/sidebar/sidebar.css
-src/components/sidebar/sidebar.jsx

-src/contexts/video-context.jsx
-src/reducers/video-reducer.jsx

-src/custom hooks/useFilteredVideos.jsx
-src/pages/home/home.jsx

Kindly review this PR and give your valuable feedback.
[Live site](https://video-library-dgt5spuqt-vrinda-mahajan.vercel.app/)

### CODE REVIEW

**Review by foolhardy21 — APPROVED** [review]
I liked that you have used `map` over  `sideBarList`. good use custom hooks for 'title'.

**Comment by vrinda-mahajan** [comment]
> I liked that you have used `map` over `sideBarList`. good use custom hooks for 'title'.

Thanks, @foolhardy21 for reviewing the PR.

File: src/components/sidebar/sidebar.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    padding-left: 2.8rem;
+    text-decoration: none;
+}
+
+.active-sidebar-item:hover {
+    background-color: var(--v1-bg-color);
+}
+
+.sidebar-item:focus {
+    background-color: var(--v1-bg-color);
```
Comment (ApurvaSawant11) [inline src/components/sidebar/sidebar.css:25]: You can merge these two `.sidebar-item:focus, sidebar-item:focus`

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me. The code was clean and very easy to understand.

File: src/components/sidebar/sidebar.css (hunk 1 above)
Comment (vrinda-mahajan) [reply src/components/sidebar/sidebar.css:25]: Thanks, @ApurvaSawant11  for pointing this out.

**Pull request merged**